tests/xtimer_drift/main.c

```diff
@@ -89,6 +89,7 @@ void *worker_thread(void *arg)
     (void) arg;
     unsigned int loop_counter = 0;
     uint32_t start = 0;
+    uint32_t last = 0;
```

Comment:
`xtimer_now` returns `uint32_t` while the loop counter doesn't really need to be any particular size, I guess I could change it if you want to.